src/client/applications/turnilo-application/turnilo-application.tsx

```diff
@@ -79,10 +83,19 @@ export class TurniloApplication extends React.Component<TurniloApplicationProps,
       selectedItem: null,
       viewType: null,
       viewHash: null,
-      showAboutModal: false
+      showAboutModal: false,
+      error: null
     };
   }
 
+  componentDidCatch(error: Error) {
+    reportError(error);
+    this.setState({
+      viewType: ERROR,
+      error: error.message
```

Comment:
So if an error happens, I will get only error message?

How about stack trace of where error happened? I cannot live without it!

Reply:
https://reactjs.org/docs/react-component.html#componentdidcatch - there is some stack trace provided by React as second parameter.

Reply:
It just seems easier flow to me (just see stack trace and immediately go to line that caused problems vs go to sentry, find logs etc. But I never used sentry. Maybe it's amazing and will make debugging easier. If so, ignore my comments.